cachi2/core/package_managers/generic/models.py

```diff
@@ -1,18 +1,25 @@
 import re
+import urllib
+from abc import ABC, abstractmethod
 from collections import Counter
+from functools import cached_property
 from pathlib import Path
-from typing import Literal
+from typing import Literal, Union
 from urllib.parse import urlparse
 
+from packageurl import PackageURL
 from pydantic import AnyUrl, BaseModel, ConfigDict, field_validator, model_validator
 from pydantic_core.core_schema import ValidationInfo
 
 from cachi2.core.checksum import ChecksumInfo
 from cachi2.core.errors import PackageManagerError
+from cachi2.core.models.sbom import Component, ExternalReference
 from cachi2.core.rooted_path import RootedPath
 
 CHECKSUM_FORMAT = re.compile(r"^[a-zA-Z0-9]+:[a-zA-Z0-9]+$")
 
+ARTIFACT_TYPES = Literal["maven"]
```

Comment:
this constant is not used anywhere

Reply:
Good catch, that was a leftover from a previous approach. Removed.